modules/svar_master/1.0/svar_master.smk

```diff
@@ -0,0 +1,251 @@
+localrules:
+    _svar_master_input_gridss,
+    _svar_master_input_manta,
+    _svar_master_output_native, 
+    _svar_master_output_lifted,
+    _svar_master_all
+
+
+##### MODULE CONFIGFILES #####
+
+# Load default module configs and update with values from svar_master config
+
+min_module_versions = {
+    "gridss": "2.0", 
+    "manta": "2.0", 
+    "hmftools": "1.1", 
+    "liftover": "2.0"
+}
+
+for tool in CFG_SV["module_versions"].keys():
+    # Ensure specified module version meets minimum requirements
+    assert float(CFG_SV["module_versions"][tool]) >= float(min_module_versions[tool]), (
+        f"The current {tool} module version is {CFG_SV['module_versions'][tool]}. "
+        f"SVAR_MASTER requires {tool} module version {min_module_versions[tool]} or higher. "
+    )
+    lcr_modules = config["lcr-modules"]["_shared"]["lcr-modules"] + "modules/"
+    configfile: lcr_modules + "/" +  tool + "/" + CFG_SV["module_versions"][tool] + "/config/default.yaml"
+    snakemake.utils.update_config(
+        config["lcr-modules"][tool], 
+        config["lcr-modules"]["svar_master"][tool]
+    )
+    if not tool == "liftover": 
+        config["lcr-modules"][tool]["inputs"]["sample_bam"] = CFG_SV["inputs"]["sample_bam"]
+        config["lcr-modules"][tool]["inputs"]["sample_bai"] = CFG_SV["inputs"]["sample_bai"]    
+
+
+##### MODULE SNAKEFILES #####
+
+# Run Manta and GRIDSS 
+include: "../../manta/" + CFG_SV["module_versions"]["manta"] + "/manta.smk"
+include: "../../gridss/" + CFG_SV["module_versions"]["gridss"] + "/gridss.smk"
+
+# Update hmftools config to use GRIDSS outputs
+config['lcr-modules']['hmftools']['inputs']['slms3_vcf'] =  CFG_SV["inputs"]["slms_3_dir"] + "/99-outputs/vcf/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.slms-3.final.vcf.gz"
+config['lcr-modules']['hmftools']['inputs']["gridss_somatic"] = str(rules._gridss_output_somatic_vcf.output.somatic)
+config['lcr-modules']['hmftools']['inputs']["gridss_somatic_tbi"] = str(rules._gridss_output_somatic_vcf.output.somatic_tbi)
+config['lcr-modules']['hmftools']['inputs']["gridss_somatic_filtered"] = str(rules._gridss_output_somatic_vcf.output.filtered)
+config['lcr-modules']['hmftools']['inputs']["gridss_somatic_filtered_tbi"] = str(rules._gridss_output_somatic_vcf.output.filtered_tbi)
+
+# Run hmftools 
+include: "../../hmftools/" + CFG_SV["module_versions"]["hmftools"] + "/hmftools.smk"
+
+##### Intersect Manta and GRIDSS #####
+
+rule _svar_master_input_gridss:
+    input:
+        vcf = str(rules._gridss_output_somatic_vcf.output.filtered), 
+        tbi = str(rules._gridss_output_somatic_vcf.output.filtered_tbi)
+    output:
+        vcf = CFG_SV["dirs"]["inputs"] + "sv_vcfs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.gridss.somatic_filtered.vcf.gz", 
+        tbi = CFG_SV["dirs"]["inputs"] + "sv_vcfs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.gridss.somatic_filtered.vcf.gz.tbi"
+    group: 
```

Comment:
Sorry, just a bit confused here. Seems that there are 2 rules combined in this group, and both are just symlinking steps. Is it meant to be bundled with some other rule that I missed, or maybe it is meant to bundle with part of gridss/hmftools pipelines?

Reply:
Ack! That's a new feature of the cookie cutter that I completely missed. Will fix.

Reply:
oh right, this is from the cookiecutter to show users how to group rules, I also forgot about this!